pyanaconda/modules/payloads/payload/rpm_ostree/installation.py

```diff
@@ -509,7 +513,7 @@ def _move_grub_config(self):
             os.rename(boot_grub2_cfg, target_grub_cfg)
             os.symlink('../loader/grub.cfg', boot_grub2_cfg)
 
-    def _set_kargs(self):
+    def _set_kargs(self, bootloader):
```

Comment:
If you add this option, please add it to docstring, it's a "proxy to bootloader". However, as it's just a Pythonified wrapper for D-Bus, it's equal to grabbing it anew as it was below, so just not doing this change is an option too.

Reply:
At a procedural level please feel free to just force-push fixups. I haven't yet spun up an Anaconda dev/test environment.